app/lib/ticket_creator.rb

```diff
@@ -1,4 +1,7 @@
 class TicketCreator
+  NORMAL_PRIORTIY = "normal".freeze
+  APP_DEFAULT_TAGS = %w[govuk_app].freeze
```

Comment:
Maybe since this is so simple and specific to the two app tickets, it should just be written out in the two app ticket creator classes.

Reply:
Good point, moved to the app ticket creators.